AgriParcel/schema.json

```diff
@@ -1,162 +1,180 @@
+        {"properties": {
+            "type": {
+                "type": "string",
+                "enum": ["AgriParcel"],
+                "description": "Property. NGSI Entity Type. It has to be AgriParcel"
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
-            }
-          ],
-          "description": "Relationship. Entity the item belongs to"
-        },
-        "ownedBy": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+            "area": {
+                "type": "number",
+                "minimum": 0,
+                "description": "Property. Model:'http://schema.org/Number'. Units:'m2'. The area of the parcel nominally in square meters."
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
-            }
-          ],
-          "description": "Relationship. Owner (Person or Organization) of the item"
-        },
-        "hasAgriParcelParent": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+            "category": {
+                "type": "string",
+                "description": "Property. Model:'http://schema.org/Text'. The category of the parcel of land e.g.: **arable, grassland, vineyard, orchard, mixed crop, lowland, upland, set-aside, forestry, wetland.**"
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
-            }
-          ],
-          "description": "Relationship. Reference to the parent AgriParcel"
-        },
-        "hasAgriParcelChildren": {
-          "type": "array",
-          "description": "Relationship. Related sub AgriParcel records to which this entity relates",
-          "items": {
-            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/EntityIdentifierType"
-          }
-        },
-        "hasAgriCrop": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+            "belongsTo": {
```

Comment:
could be more explicit the difference between belongsto and ownedBy? (just extend the descriptions)